client/src/featureform/proto/metadata.proto

```diff
@@ -69,6 +69,7 @@ message ResourceStatus {
         PENDING = 2;
         READY = 3;
         FAILED = 4;
+        READY_ONLINE = 5;
```

Comment:
Do we need another status for if offline is ready?

Currently if offline store succeeds, but online store fails, then it counts the whole job as failed. However the training set will still run.

Reply:
Maybe have a FAILED_ONLINE status so we know that the offline table was created so it doesnt linger

Reply:
honestly it makes sense to have status for online and offline separately since the logic is so bifurcated